components/event-banner/EventBanner.vue

```diff
@@ -1,8 +1,21 @@
 <template functional>
   <div class="event-banner">
     <div class="event-banner__banner">
-      <h2 class="event-banner__title">
-        {{ props.location }} {{ props.name }}
+      <img
+        v-if="props.bannerIcon"
+        :src="props.bannerIcon.url"
+        :width="props.bannerIcon.width"
+        :height="props.bannerIcon.height"
+        alt=""
+        class="event-banner__icon"
+      >
+      <h2
+        class="event-banner__title"
+        :class="{
+          'sr-only': props.visuallyHideName
```

Comment:
Why not base the 'sr-only' on `bannerIcon`?

Reply:
Because I want to give them the option to both show the title and the icon.